src/ui/server/controllers/insurance/declarations/confirmation-and-acknowledgements/index.ts

```diff
@@ -52,25 +51,16 @@ export const get = async (req: Request, res: Response) => {
     return res.redirect(PROBLEM_WITH_SERVICE);
   }
 
-  try {
-    const declarationContent = await api.keystone.application.declarations.getLatestConfirmationAndAcknowledgement();
-
-    return res.render(TEMPLATE, {
-      ...insuranceCorePageVariables({
-        PAGE_CONTENT_STRINGS: CONFIRMATION_AND_ACKNOWLEDGEMENTS,
-        BACK_LINK: req.headers.referer,
-      }),
-      ...pageVariables(application.referenceNumber),
-      userName: getUserNameFromSession(req.session.user),
-      documentContent: declarationContent.content.document,
-      documentConfig: keystoneDocumentRendererConfig(),
-      application: mapApplicationToFormFields(res.locals.application),
-    });
-  } catch (err) {
-    console.error("Error getting declarations - confirmation and acknowledgements and rendering 'confirmation and acknowledgements' page %O", err);
-
-    return res.redirect(PROBLEM_WITH_SERVICE);
-  }
+  return res.render(TEMPLATE, {
+    ...insuranceCorePageVariables({
+      PAGE_CONTENT_STRINGS: CONFIRMATION_AND_ACKNOWLEDGEMENTS,
+      BACK_LINK: req.headers.referer,
+    }),
+    ...pageVariables(application.referenceNumber),
+    userName: getUserNameFromSession(req.session.user),
+    CONTENT: CONFIRMATION_AND_ACKNOWLEDGEMENTS,
```

Comment:
Similar to above - having `PAGE_CONTENT_STRINGS` and `CONTENT` referencing the same thing is strange, can it just be `PAGE_CONTENT_STRINGS`?